pgModel/models.go

```diff
@@ -0,0 +1,65 @@
+package pgModel
+
+type JwtToken struct {
+    Token string `json:"token"`
+}
+
+type Exception struct {
+    Message string `json:"message"`
+}
+
+type User struct {
+	Id     int64 `sql:",pk,unique"`
+	Username   string
+	Email string `sql:",unique,notnull"`
+	Password string
+	PhoneNumber string
+	Messages []*Message
+	Connections []*Connection
+	Comments []*Comment
+	Collections []*Collection
+}
+
+type Connection struct {
+	Id int64
+	FirstUserId int64
+	SecondUserId int64
+}
+
+type Message struct {
+	Id int64
+	UserId int64
+	Content string
+	MessageSender string
```

Comment:
@mikey2020 I see UserId(int) and MessageSender(string) in the Message struct.
Are they meant to store different values? 
and what is the use case for having both?

Reply:
Its the model for sending messages between two users